tests/foreman/cli/test_satellitesync.py

```diff
@@ -64,13 +64,58 @@ def config_export_import_settings():
     Settings.set({'name': 'content_disconnected', 'value': disconnected_mode_value})
 
 
+@pytest.fixture(scope='function')
+def export_cleanup_function(default_sat, function_org):
+    """Deletes export/import dirs of function org"""
+    yield
+    # Deletes directories created for export/import test
+    assert default_sat.execute(f'rm -rf {EXPORT_DIR}/{function_org.name}').stdout == ''
+    assert default_sat.execute(f'rm -rf {IMPORT_DIR}/{function_org.name}').stdout == ''
+
+
+@pytest.fixture(scope='function')
```

Comment:
```suggestion
@pytest.fixture(scope='function')
```
Is that the right scope?
If it is, could you have just one fixture and pass in the org name using "request" to get it?
This is non-blocking comment.

Reply:
I don't think this can be flattened down to one fixture - but I think you're right that the scope of this fixture should me `module`.

Reply:
That scope is on purpose. It should cleanup the export/import directories after each testcase(function) of same module.
I know it might look like a hack, I'll try to come up with some better implementation for 6.11.

Reply:
Gotcha! Can you add an inline comment so its clear (in the future) that is the intent here. I don't think its too much of a hack, maybe its not elegant but its explicit and not a widespread pattern.